src/org/chartjs/CandlestickDAOChartView.js

```diff
@@ -75,14 +77,28 @@ foam.CLASS({
       class: 'foam.mlang.ExprProperty',
       name: 'yExpr',
       factory: function() { return this.Candlestick.AVERAGE; }
+    },
+    {
+      class: 'Int',
+      name: 'width',
+      factory: function() {
```

Comment:
Why factory instead of just doing value: 750?

Reply:
true. will change.